ui/src/data-products/element-list/location-track-element-listing-search.tsx

```diff
@@ -214,22 +221,27 @@ const LocationTrackElementListingSearch = ({
                         ).map((error) => t(`data-products.search.${error}`))}
                     />
                 </div>
-                <Button
-                    className={styles['element-list__download-button']}
-                    disabled={!state.elements || state.elements.length === 0}
-                    onClick={() => {
-                        if (state.searchParameters.locationTrack) {
-                            location.href = getLocationTrackElementsCsv(
-                                state.searchParameters.locationTrack?.id,
-                                selectedElementTypes(state.searchParameters.searchGeometries),
-                                validTrackMeterOrUndefined(state.searchParameters.startTrackMeter),
-                                validTrackMeterOrUndefined(state.searchParameters.endTrackMeter),
-                            );
+                <a
```

Comment:
Linkki jonka sisällä on nappi on rakenteena aika nurinkurinen. Jos tälle muutokselle oli joku selkeä tarve, niin voisi sitten vaan mennä all the way ja käyttää ihan pelkkää linkkiä